### PR TITLE
Hide empty block

### PR DESCRIPTION
**Pull request by hankxdev**

### CODE REVIEW

File: src/components/BlockInfo.vue [hunk 1]
```diff
@@ -9,7 +9,7 @@
       <component :is="txInfoComponent" :tx="selectedTx.data"/>
     </div>
     <div v-if="!isLoading && !selectedTx">
-      <div v-if="block" class="text-muted">Created by <span class="node-name">{{ nodeName }}</span></div>
+      <!-- <div v-if="block" class="text-muted">Created by <span class="node-name">{{ nodeName }}</span></div> -->
```
Comment (mattkanwisher) [inline src/components/BlockInfo.vue]: lets remove this code

File: src/components/block-list.ts [hunk 2]
```diff
@@ -35,7 +36,8 @@ export default class BlockList extends Vue {
   sortDesc = true
   fields = [
     { key: 'blockHeight', label: 'Block', sortable: true },
-    { key: 'numTransactions', label: 'Txs Processed', sortable: true },
+    // { key: 'numTransactions', label: 'Txs Processed', sortable: true },
```
Comment (mattkanwisher) [inline src/components/block-list.ts]: lets remove this code

**Comment by mattkanwisher** [comment]
Remove the two commented lines then feel free to merge

File: src/components/block-list.ts (hunk 2 above)
Comment (hankxdev) [reply src/components/block-list.ts]: removed

**Pull request merged**